config/locales/en.yml

```diff
@@ -378,3 +378,6 @@ en:
                   mathematics: "Select yes if you currently teach mathematics"
                   foreign_languages: "Select yes if you currently teach foreign languages"
                   physics: "Select yes if you currently teach physics"
+  levelling_up_payments:
+    policy_short_name: "Levelling Up Payments"
+    policy_name: "Claim a levelling up payment"
```

Comment:
This won't get used, just need the short one, shouldn't break anything 🤞 

`policy_name` is actually the name of the "service" under "Manage Services" and in turn is the text that appears in the black head banner and we're renaming the ECP service to the ECP&LUP service.

Reply:
`policy_name`'s gone now, thanks